src/sections/about.html

```diff
@@ -0,0 +1,16 @@
+<div>
+  <p class="font-body font-normal text-black text-2xl lg:text-4xl md:text-3xl px-8 md:px-16 lg:px-24 ">
+    We are a Berlin-based communications agency specialising in helping
+    companies and brands find their voice and position themselves. Whether it's
+    brand building, identity sharpening, or communication strategy, we bring a
+    fresh perspective to established structures, develop tailor-made
+    communication solutions, and ensure that messages resonate. With over a
+    decade of experience, we have honed our skills in giving brands their
+    unmistakable identity and making them visible in the long term.
+  </p>
+  <div class="flex justify-center mt-16 items-center sticky top-0">
+    <img src="/img/Vector-12.webp">
```

Comment:
Can you add the `alt` in case the image is not displayed to show the alt value of the image? 
 `<img src="/img/Vector-12.webp" alt ="image name">`